README.md

```diff
@@ -57,9 +65,29 @@ app.register_function(sample_page)
 # Run the server
 server = app.run()
 ```
+## Whos is PyCob for?
+
+PyCob is perfect for quant and analytical developers who want to expose data analysis.  PyCob's sweetspot sits between jupyter notebooks and apps that are supported by a full frontend dev team.  
+
+
+### Scaling an app
```

Comment:
This is a very rough pass explaining where PyCob fits into different deployment schemes.  I think actually adding this info and discussing competitors establishes you as an expert vs talking about your solution in a vacuum.